app/routes/endemics/herd-others-on-sbi.js

```diff
@@ -4,30 +4,47 @@ import links from '../../config/routes.js'
 import { sessionKeys } from '../../session/keys.js'
 import { getEndemicsClaim, setEndemicsClaim } from '../../session/index.js'
 import HttpStatus from 'http-status-codes'
+import { OTHERS_ON_SBI } from '../../constants/herd.js'
+import { getHerdOrFlock } from '../../lib/display-helpers.js'
 
 const { urlPrefix } = config
 const {
   endemicsHerdOthersOnSbi,
   endemicsEnterCphNumber,
-  endemicsEnterHerdDetails
+  endemicsEnterHerdDetails,
+  endemicsCheckHerdDetails
 } = links
 
 const pageUrl = `${urlPrefix}/${endemicsHerdOthersOnSbi}`
 const previousPageUrl = `${urlPrefix}/${endemicsEnterCphNumber}`
-const nextPageUrl = `${urlPrefix}/${endemicsEnterHerdDetails}`
+const enterEnterHerdDetailsPageUrl = `${urlPrefix}/${endemicsEnterHerdDetails}`
+const checkHerdDetailsPageUrl = `${urlPrefix}/${endemicsCheckHerdDetails}`
+const ONLY_HERD = 'onlyHerd'
 
-const { endemicsClaim: { herdOthersOnSbi: herdOthersOnSbiKey } } = sessionKeys
+const { endemicsClaim: { herdOthersOnSbi: herdOthersOnSbiKey, herdReasons: herdReasonsKey } } = sessionKeys
+
+const getSpeciesGroupText = (typeOfLivestock) => {
+  const textByLivestock = {
+    beef: 'beef cattle herd',
+    dairy: 'dairy cattle herd',
+    pigs: 'pigs herd',
+    sheep: 'flock of sheep'
+  }
+  return textByLivestock[typeOfLivestock] || ''
```

Comment:
do we ever want to return an empty string? With how this function is used, I dont think an empty string will look right.

Reply:
yeah or even just always `return textByLivestock[typeOfLivestock]`